pkgs/development/ocaml-modules/angstrom/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchFromGitHub, ocaml, cstruct, ocplib-endian, result, findlib, ocaml_oasis }:
+
+stdenv.mkDerivation rec {
+  version = "0.3.0";
+  name = "ocaml-angstrom-${version}";
+
+  src = fetchFromGitHub {
+    owner  = "inhabitedtype";
+    repo   = "angstrom";
+    rev    = "${version}";
+    sha256 = "1x9pvy5vw98ns4pspj7i10pmgqyngn4v4cdlz5pbvwbrpwpn090q";
+  };
+
+  createFindlibDestdir = true;
+
+  buildInputs = [ ocaml ocaml_oasis findlib ocplib-endian ];
+  propagatedBuildInputs = [ result cstruct ];
+
+  meta = {
+    homepage = https://github.com/inhabitedtype/angstrom;
+    description = "Parser combinators built for speed and memory efficiency";
+    license = stdenv.license.bsd3;
```

Comment:
There is no `stdenv.license` attribute, but there is a `stdenv.lib.licenses` one.